flips/20220708-resource-reference-invalidation.md

```diff
@@ -0,0 +1,101 @@
+# Invalidate References to Transferred Resources
+
+| Status        | Proposed                                     |
+:-------------- |:---------------------------------------------|
+| **FLIP #**    | N/A                                          |
+| **Author(s)** | Supun Setunga (supun.setunga@dapperlabs.com) |
+| **Sponsor**   | Supun Setunga (supun.setunga@dapperlabs.com) |
+| **Updated**   | 2022-07-08                                   |
+
+## Objective
+
+This proposal is to introduce an invalidation for ephemeral reference values, if the referring resource is transferred
```

Comment:
Maybe mention here and also in the title that this proposal is for Cadence. 

Also, it might not be clear to users what transfer means (it is not described in the documentation), so maybe explain it in 1-2 lines.